package.json

```
@@ -79,7 +79,7 @@
 		"test:e2e:update": "npm run wp-env:config && cross-env JEST_PUPPETEER_CONFIG=tests/e2e/config/jest-puppeteer.config.js NODE_CONFIG_DIR=tests/e2e/config wp-scripts test-e2e --config tests/e2e/config/jest.config.js --updateSnapshot",
 		"test:help": "wp-scripts test-unit-js --help",
 		"test:performance": "npm run wp-env:config && cross-env NODE_CONFIG_DIR=tests/e2e/config wp-scripts test-e2e --config tests/e2e/config/jest.performance.config.js -- performance",
-		"test:php": "npm run wp-env run phpunit '/var/www/html/wp-content/plugins/$(basename $(pwd))/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/$(basename $(pwd))/phpunit.xml.dist'",
+		"test:php": "npm run wp-env run phpunit '/var/www/html/wp-content/plugins/$(basename \"$(pwd)\")/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/$(basename \"$(pwd)\")/phpunit.xml.dist'",
```

Hey @nielslange previous fix does not work, and there's no need to use basename here. Still I feel you can never really win with these things, we just keep trading bugs 🙄😂

```suggestion
		"test:php": "npm run wp-env run phpunit '/var/www/html/wp-content/plugins/${PWD##*/}/vendor/bin/phpunit -c /var/www/html/wp-content/plugins/${PWD##*/}/phpunit.xml.dist'",
```

---

> Hey @nielslange previous fix does not work, and there's no need to use basename here. Still I feel you can never really win with these things, we just keep trading bugs 🙄😂

Thanks for the suggestions. Can you think of a better approach to solve the current problem?

---

That is the best alternative I can think of 😅

---

Given that this is the best alternative you can think of, I'm happy to accept your suggestion, @wavvves! 🙏